Lykke.WalletApiv2.Tests/SettingsTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Diagnostics;
+using System.Net;
+using Core;
+using Core.Settings;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Lykke.WalletApiv2.Tests
+{
+    public class SettingsTest
+    {
+        [Fact]
+        public void Cleanup()
+        {
+            var data = new WebClient().DownloadString("http://settings.lykke-settings.svc.cluster.local/rr5999apiv2999dvgsert25uwheifn_WalletApiv2");
```

Comment:
I'm not sure hardcoding url-s into github repository is a good idea, even though it's an internal url.